packages/common/sagas/entities/item.js

```diff
@@ -698,19 +698,19 @@ export function* createItemsBatchSaga({
       }),
     );
 
-    const { data } = yield call(postCreateItemsBatch, {
+    const { data: serverItems } = yield call(postCreateItemsBatch, {
       items: preparedForRequestItems,
     });
 
-    const preparedForStoreItems = data.map((item, index) => ({
+    const preparedForStoreItems = serverItems.map((item, index) => ({
       ...item,
       data: preparedForEncryptingItems[index],
     }));
+    const { itemsById } = convertItemsToEntities(preparedForStoreItems);
+    const itemIds = Object.keys(itemsById);
 
-    yield put(createItemsBatchSuccess(preparedForStoreItems));
-    yield put(addItemsBatchToList(data.map(({ id }) => id), listId));
-    yield put(updateGlobalNotification(NOOP_NOTIFICATION, false));
-
+    yield put(createItemsBatchSuccess(itemsById));
+    yield put(addItemsBatchToList(itemIds, listId));
```

Comment:
The action `addItemsBatchToList` is deprecated due to we don't use the children anymore. Each item must have a listId as a field inside itself.

Reply:
yep. I know. 
but for now if it does not added to list-children importing does not works correctly.

we need to plan refactoring for children in lists. there are a lot of actions using it.